README.md

```diff
@@ -4,6 +4,12 @@ The [etcd.io][] website, built using [Hugo][] and hosted on [Netlify][].
 
 ## Build prerequisites
 
+### Cloud build
+
+Visit [https://gitpod.io/#https://github.com/etcd-io/website](https://gitpod.io/#https://github.com/etcd-io/website) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
+
+### Local build
+
```

Comment:
IMHO, you can drop this addition and instead edit the level-2 heading to read:

> ## Local build

If you do so, then avoid the subheading promotions below.